ansible/include_vars.yml

```diff
@@ -1,65 +1,45 @@
 ---
-- name: Step 0000 Find Include Vars files
-  hosts: localhost
-  connection: local
+- name: Step 0000 Include Vars
+  hosts:
+  - localhost
+  - all
   gather_facts: no
   tags: include_vars
   tasks:
-  - name: Stat default variables files (both yaml/yml extensions)
+  - name: Set output_dir for all hosts
+    set_fact:
+      output_dir: "{{ hostvars.localhost.output_dir }}"
+    when: hostvars.localhost.output_dir is defined
+
+  - name: Include variables files
     vars:
-      find_me:
+      __vars_file_base:
       # Global default vars related to the cloud provider
       - cloud_providers/{{ cloud_provider }}_default_vars
       # Legacy env_vars.yml (replaced by default_vars.yml)
       - configs/{{ env_type }}/env_vars
       # Default vars of the config
       - configs/{{ env_type }}/default_vars
       # Default vars of the config, specific to a cloud provider
-      - configs/{{ env_type }}/default_vars.{{ cloud_provider }}
       - configs/{{ env_type }}/default_vars_{{ cloud_provider }}
       # Lecacy secret vars file.
       - configs/{{ env_type }}/env_secret_vars
-      extensions:
+      __vars_file_extension:
       - yaml
       - yml
-    stat:
-      path: "{{ item[0] ~ '.' ~ item[1] }}"
-    loop: "{{ find_me | product(extensions) | list }}"
-    register: rstat_varfiles
-
-  - name: Stat variables files
-    vars:
-      find_me:
-      # secret file path passed as extra-var
-      - "{{ secret_file | d('/secret/file/not/passed') }}"
```

Comment:
Can you keep this feature of `secret_file`?
I'm not sure we're using it but your change can break that

Reply:
Re added this functionality.